docs/email_formatting.md

```diff
@@ -0,0 +1,73 @@
+# Email Formatting
+
+This team takes emails that have been reviewed by the [Email Review Team](/docs/email_reviewing.md), formats them into code, and then submits a Pull Request (PR) to add the email to the website. It is semi-technical, requiring a basic understanding of GitHub, markdown, YAML, and coding style. First of all, you will need to make a GitHub Account. Then you will need some permissions to edit the labels on issues. Submit an issue to request that [here](https://github.com/defund12/defund12.org/issues/new?assignees=&labels=admin-attention&template=TRIAGE_ACCESS.md&title=Triage+access+for+%5BUSERNAME%5D).
+
+You can talk to other members of the team on the corresponding channel on [our Discord server](https://discord.gg/ZjSw6mb).
+
+## Find an available issue
+
+Find [issues](https://github.com/defund12/defund12.org/issues?q=is%3Aissue+is%3Aopen+label%3Aready-for-pr) with the <kbd>ready-for-pr</kbd> label. Claim the issue in a comment or assign it to yourself so no one else begins working on it.
+
+## Create a new template file
+
+- Create a new `country/state/city.md` file in the `/_emails` directory.
+- Copy an existing `.md` template and modify it for the new email.
+- Ensure it is properly formatted. Here is an example of a bad email `.md` file:
+```
+title: Manchester, New Hampshire
```

Comment:
Re: quotes — there was a [convo in Discord today](https://discordapp.com/channels/718581500148449292/718920528806674552/719990042709196900) encouraging  email devs to wrap text in quotes. Other punctuation could trip this up too, I think? Maybe just `-` hyphens tho. 

The lift is much less of a concern since @rpalakkal made an incredible lil app that autoformats after pasting! https://emailaddressformat.rpal.repl.run/

Reply:
Re: quotations, OK, I'll defer that to you. If the linter and tests catch it then it's fine. I figured consistency was more important, but you're the one chugging away at that task, so...

Re: 2-space indent, do you see that I had that as a note?